### PR TITLE
Added ability to cycle through all vrf on iosxe

### PR DESCRIPTION
**Pull request by jbronikowski**

### CODE REVIEW

**Comment by LukasMcClelland** [comment]
Hello @jbronikowski, thank you very much for the contribution! 

Unfortunately, we were having an issue with our automatic Github Actions testing, but thankfully, that issue has now been fixed. This means that your branch must be updated to include that fix. 

Please do one of the following options (depending on your knowledge of git):
  1. Merge your branch with the latest version of `master` https://github.com/CiscoTestAutomation/genielibs 
  OR
  2. Since you've only changed one file: make a new fork of this repo, make your code changes, and open a new pull request and select me as a reviewer (easier option if you're newer to git)
  
My apologies for the inconvenience. Please let me know if you need any help!

Lukas

**Comment by LukasMcClelland** [comment]
Hello @jbronikowski, do you plan on finishing this PR soon? If not, that's OK, we will fix it up and merge it for you

File: pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py [hunk 1]
```diff
@@ -18,67 +18,86 @@ def keys(self, item):
             return list(item.keys())
         return []
 
-    def learn(self, address_family=None, vrf=None, protocol=None, route=None, interface=None):
```
Comment (tahigash) [inline pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py:21]: I think no need to remove `vrf`. if None, will get all vrfs, if specified, only that vrf.

Comment (LukasMcClelland) [reply pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py:21]: vrf added back in

Comment (tahigash) [reply pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py:21]: codes looks good. but could you please check if UTs are enough to test both all VRFs and specific vrf? if not enough, please update UTs.

Comment (tahigash) [reply pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py:21]: @LukasMcClelland , just reminder.

Comment (LukasMcClelland) [reply pkgs/ops-pkg/src/genie/libs/ops/routing/iosxe/routing.py:21]: Updated tests. Ready for re-review

**Review by tahigash — APPROVED** [review]

**Review by LukasMcClelland — APPROVED** [review]

**Pull request merged**